### ISSUE TITLE
[Question] ...

### ISSUE THREAD

**Issue by aeaeughugh** [SuperMartijn642/ChunkLoaders#90]
**Question**
Are placed chunk loaders supposed to be invisible? Whenever I place a chunk loader (I only tried this with the 1x1 and 3x3 chunk loaders) they appear as invisible, and when I mine them only the block-breaking cracks appear.

**Comment by SuperMartijn642** [comment]
That seems like a bug. What version of Minecraft and Chunk Loaders are you using?

I'd guess it is caused by some other mod, could you narrow down which other mods are needed to reproduce the issue?

**Comment by aeaeughugh** [comment]
It

**Comment by aeaeughugh** [comment]
> That seems like a bug. What version of Minecraft and Chunk Loaders are you using?
> 
> I'd guess it is caused by some other mod, could you narrow down which other mods are needed to reproduce the issue?

It appears that the mod "Chunk Animator" does this. As soon as I removed it, the chunk loader blocks started to appear normally. I was using ChunkAnimator-1.20.1-1.3.7. I've attached the mod file as a zip, since github doesn't support .jar files which was the original file type.

[ChunkAnimator-1.20.1-1.3.7.zip]
(https://github.com/SuperMartijn642/ChunkLoaders/files/11928885/ChunkAnimator-1.20.1-1.3.7.zip)
